stock-usecase/src/main/java/org/dhis2/android/rtsm/ui/home/screens/components/DropdownComponents.kt

```diff
@@ -57,7 +58,6 @@ import org.dhis2.commons.orgunitselector.OrgUnitSelectorScope
 import org.hisp.dhis.android.core.option.Option
 import org.hisp.dhis.android.core.organisationunit.OrganisationUnit
 
-var orgUnitData: OrganisationUnit? = null
 var orgUnitName: String? = null
```

Comment:
I think that you can remove this propperty as well, inside the settingsUiState you have a function called facilityName